src/workloads/scale/LargeScaleLongLived.yml

```diff
@@ -0,0 +1,82 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
+
+Actors:
+- Name: LongLivedCreator
+  Type: Loader
+  Threads: 10
+  Phases:
+  - Repeat: 1
+    Database: &DB longlived
+    CollectionCount: &CollectionCount 1000
+    Threads: 10
+    DocumentCount: &DocumentCount 1000
+    BatchSize: 1000
+    Document:
+      # Each document ranges in size from about 90 to 150 bytes (average 120)
+      x0: &rand_10k_int {^RandomInt: {min: 0, max: 10000}}
+      x1: &rand4int {^RandomInt: {min: 0, max: 2147483647}}
+      x2: *rand4int
+      x3: *rand4int
+      x4: *rand4int
+      x5: *rand4int
+      x6: *rand4int
+      x7: *rand4int
+      x8: *rand4int
+      s0: {^RandomString: {length: {^RandomInt: {min: 20, max: 80}}}}
+    Indexes:
+    - keys: {x0: 1}
+    - keys: {x1: 1}
+    - keys: {x2: 1}
+    - keys: {x3: 1}
+    - keys: {x4: 1}
+    - keys: {x5: 1}
+    - keys: {x6: 1}
+    - keys: {x7: 1}
+    - keys: {x8: 1}
+  - {Nop: true}
+
+- Name: LongLivedIndexReader
+  Type: MultiCollectionQuery
+  Threads: 100
+  GlobalRate: 10000 per 1 second
+  Phases:
+  - {Nop: true}
+  - Duration: 10 minutes
+    Database: *DB
+    CollectionCount: *CollectionCount
+    DocumentCount: *DocumentCount
+    Threads: 10
```

Comment:
This thread count differs with the one on line 46. How should I be reading it?

Reply:
This was a mistake.  Having a "Threads" parameter in the actor phase only has meaning for the Loader actor (it may have something to do with how the loading work is broken up...?).  Anyway having a "Threads" parameter in the actor phase for a MultiCollection{Query,Update} is ignored, and is confusing.  I'm removing it in the 3 places I did that and doing another evergreen run to make sure it doesn't affect anything.

Reply:
Thanks